x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/PreparedStatementTestCase.java

```diff
@@ -207,4 +301,22 @@ public void testSingleParameterMultipleTypes() throws Exception {
             return result;
         }
     }
+
+    private static long convertFromCalendarToUTC(long value, Calendar cal) {
```

Comment:
Is this something used somewhere else that could be moved into a DateUtils/TestUtils of sorts?